core/src/stores/store.rs

```diff
@@ -311,9 +380,41 @@ pub const POSTGRES_TABLES: [&'static str; 11] = [
        status                   TEXT NOT NULL,
        FOREIGN KEY(data_source) REFERENCES data_sources(id)
     );",
+    "-- databases
+    CREATE TABLE IF NOT EXISTS databases (
+       id                   BIGSERIAL PRIMARY KEY,
+       created              BIGINT NOT NULL,
```

Comment:
Do you plan to share this model in the world of remote databases ?

In which case this will have a type: "local" | "remote" and a connection string right?

Or do you plan to have a different model entirely?

Reply:
I plan to have it here in the same table, but wasn't too bullish on adding stuff we won't be using right away. Do you prefer that I add this now vs migrate later on ?

Reply:
Nope just want to make sure we have a good path forward there :+1: